INSTALL.md

```diff
@@ -141,11 +141,13 @@ especially if you're using a user-managed node install such as [nvm](https://git
     ([more information](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse)).
     For other versions of Windows, there are several ssh/OpenSSH clients provided by 3rd parties.
 
-  * If you need SSH to work behind a proxy, you will also need to install
-    [`proxytunnel`](http://proxytunnel.sourceforge.net/) (available as a `proxytunnel` package
-    for Ubuntu, for example).
-    Check the [README](https://github.com/balena-io/balena-cli/blob/master/README.md) file
-    for proxy configuration instructions.
+  * The [`proxytunnel`](http://proxytunnel.sourceforge.net/) package (command-line tool) is needed
+    for the `balena ssh` command to work behind a proxy. It is available for Linux distributions
+    like Ubuntu/Debian (`apt install proxytunnel`), and for macOS through
+    [Homebrew](https://brew.sh/). Windows support is limited to the Windows Subsystem for Linux
+    (e.g., by installing Ubuntu through the Microsoft App Store). Check the
+    [README](https://github.com/balena-io/balena-cli/blob/master/README.md) file for proxy
+    configuration instructions.
```

Comment:
@richbayliss or @wrboyce, do you know if `proxytunnel` is only meant to work with openBalena? As documented by Rich on a forum post: https://forums.balena.io/t/howto-ssh-into-host-device/5312

I was testing `balena ssh` in the CLI's ***master branch*** (before this PR) with balenaCloud (not openBalena) and a proxy server (squid), but never managed to get it working when `proxytunnel` is used. Tested both Linux (Debian) and macOS, same result:
```
$ DEBUG=1 ./bin/balena ssh b1878c618483983448f6b863d30df408 -v
[debug] original argv0="node" argv=[/Users/paulo/.nvm/versions/node/v10.15.3/bin/node,/Users/paulo/balena/balena-io/balena-cli/bin/balena-dev,ssh,b1878c618483983448f6b863d30df408,-v] length=5
[Debug]   Fetching application by name b1878c618483983448f6b863d30df408 (string)
[Debug]   Application not found
[Debug]   Fetching device by UUID b1878c618483983448f6b863d30df408 (string)
vpncmd: ssh -vvv -t -o LogLevel=ERROR -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ProxyCommand=proxytunnel\ --proxy\=127\\\.0\\\.0\\\.1\\\:8888\ --dest\=\\\%h\\\:\\\%p  gh_paulo_castro@ssh.balena-devices.com host b1878c618483983448f6b863d30df408
OpenSSH_7.9p1, LibreSSL 2.7.3
debug1: Reading configuration data /Users/paulo/.ssh/config
debug1: /Users/paulo/.ssh/config line 107: Applying options for *
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 48: Applying options for *
debug1: Executing proxy command: exec proxytunnel --proxy=127\\.0\\.0\\.1\\:8888 --dest=\\ssh.balena-devices.com\\:\\22
debug1: identity file /Users/paulo/.ssh/id_rsa type 0
debug1: identity file /Users/paulo/.ssh/id_rsa-cert type -1
debug1: identity file /Users/paulo/.ssh/id_dsa type -1
debug1: identity file /Users/paulo/.ssh/id_dsa-cert type -1
debug1: identity file /Users/paulo/.ssh/id_ecdsa type -1
debug1: identity file /Users/paulo/.ssh/id_ecdsa-cert type -1
debug1: identity file /Users/paulo/.ssh/id_ed25519 type -1
debug1: identity file /Users/paulo/.ssh/id_ed25519-cert type -1
debug1: identity file /Users/paulo/.ssh/id_xmss type -1
debug1: identity file /Users/paulo/.ssh/id_xmss-cert type -1
debug1: Local version string SSH-2.0-OpenSSH_7.9
Via 127.0.0.1:8888 -> ssh.balena-devices.com:22
HTTP return code: 403 Forbidden
ssh_exchange_identification: Connection closed by remote host
```